aplib.net-demo/Assets/Scripts/Entities/Player/PlayerLogic.cs

```diff
@@ -7,6 +7,7 @@
 [RequireComponent(typeof(HealthComponent))]
 [RequireComponent(typeof(Movement))]
 [RequireComponent(typeof(RespawnableComponent))]
+[RequireComponent(typeof(Animator))]
```

Comment:
I would remove the one here, since you already added in the movement script and this one requires movement

Reply:
It wasn't here at first, but removing it here breaks one of the tests as the other script is not present within that test scene.

Reply:
All tests succeed for me without this